packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,9 @@
+{
+  "pci_databases_service_integration_tab_title": "Intégration de services",
+  "pci_databases_service_integration_tab_descritpion": "L’intégration de services vous permet de connecter différentes solutions Kafka.",
+  "pci_databases_service_integration_tab_kafka_service": "Service Kafka",
+  "pci_databases_service_integration_tab_add_kafka": "Ajouter un Kafka",
```

Comment:
```suggestion
  "pci_databases_service_integration_tab_add_kafka": "Ajouter un service Kafka",
```

Reply:
Same: I cannot change the text as it has been already validated by content team.